syz-fuzzer/proc.go

```diff
@@ -281,6 +284,7 @@ func (proc *Proc) enqueueCallTriage(p *prog.Prog, flags ProgTypes, callIndex int
 }
 
 func (proc *Proc) executeAndCollide(execOpts *ipc.ExecOpts, p *prog.Prog, flags ProgTypes, stat Stat) {
+	proc.resetAccState()
```

Comment:
Can we move it into execute instead of sprinkling everywhere where execute is called?
If we want to opt out some runs from reset, execute could accept an additional flag, or look at existing flags/stat to decide to reset or not.

Reply:
I've added an extra parameter -- we cannot determine whether to reset looking at stat only, e.g. in case of minimization we do it only once.

I also disabled reset for `WorkCandidate` -- if there was new coverage, we'd triage it properly anyway.

Reply:
> I also disabled reset for WorkCandidate -- if there was new coverage, later we'd triage it properly anyway.

Can it happen that it won't give new coverage w/o reset, but would give with a reset? I think it's possible.

Reply:
I think that even if it can be, the probability is very very low. Also, in this PR we already restart executors quite a lot, so there shouldn't be much accumulated state in the first place.